modules/openshift-developer-cli-mirroring-a-supported-builder-image.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// cli_reference/openshift_developer_cli/using_odo_in_a_restricted_environment/creating-and-deploying-a-component-to-the-disconnected-cluster
+
+[id="mirroring-a-supported-builder-image_{context}"]
+= Mirroring a supported builder image
+
+.Procedure
+
+. Verify that the required images tag is not imported:
++
+----
+$ oc describe is nodejs -n openshift
```

Comment:
Not critical, but probably would be a good idea to tell users what to expect the output of this command to be. And what should they do if it is important? Should they stop following the procedure?